config/default.yml

```diff
@@ -989,6 +989,13 @@ Metrics/PerceivedComplexity:
 Lint/AssignmentInCondition:
   AllowSafeAssignment: true
 
+Lint/BlockAlignment:
+  AlignWith: either
```

Comment:
Hmm, I forgot about this. I wonder why we did things differently there. @jonas054 thoughts?